src/features/goblins/bank/lib/bankUtils.ts

```diff
@@ -70,6 +71,15 @@ export function canWithdraw({ item, game }: CanWithdrawArgs) {
     return !cropIsPlanted({ item: "Parsnip", game });
   }
 
+  // Make sure no trees are replenishing
```

Comment:
Not a big deal, but comment doesn't match the code.

Reply:
done